source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -49,6 +49,14 @@ GlslRenderer::GlslRenderer(unsigned int width, unsigned int height) :
 
 GlslRenderer::~GlslRenderer()
 {
+    if (_context->makeCurrent())
+    {
+        if (_program->geometryBound())
```

Comment:
Minor: You can check this condition before trying to make current otherwise it's not required.